### PR TITLE
GSREN3D-155: Make information panel retractable, GSREN3D-157: map extent

### PR DESCRIPTION
**Pull request by ismailsunni**
### JIRA issue

https://jira.camptocamp.com/browse/GSREN3D-155
https://jira.camptocamp.com/browse/GSREN3D-157

#### GSREN3D-155
- Add pinia store to store the state of the information panel
- Merge the pinia store from the photo gallery with the information panel (under panelStore)
- Bind visibility of the information panel to the pinia store
- Add a button to toggle the information panel visibility

#### GSREN3D-157
- Adjust map extent so that it doesn't change when the information panel is shown/hidden

### Screenshots

Peek 2022-10-14 12-17 - retractable panel

### CODE REVIEW

File: src/components/home/InformationPanel.vue
Comment (fgravin) [inline src/components/home/InformationPanel.vue:1]: The collapsible behavior should be handled in another higher component, because it's shared to all pages.

File: src/components/home/InformationPanel.vue
Comment (fgravin) [inline src/components/home/InformationPanel.vue:22]: I think they will ask for an animation, therefore, using the `hidden` class will not work. You should play with size change transition, or maybe a translation.

**Review by fgravin — CHANGES_REQUESTED** [review]
Good work but it requires some refactoring and some adaptation on the way to collapse the panel.
Thanks

**Comment by fgravin** [comment]
@ismailsunni you can get inspiration from https://github.com/camptocamp/geoide-entrepot-ui/pull/10/commits/1e5affd80b565aa3e2f903641f327b50ec98690b

File: src/components/home/InformationPanel.vue
Comment (fgravin) [inline src/components/home/InformationPanel.vue:21]: should have a fixed width

File: src/views/HomeView.vue
Comment (fgravin) [inline src/views/HomeView.vue:8]: `<main>` should not be flex anymore, but relative.

File: src/views/HomeView.vue
Comment (fgravin) [inline src/views/HomeView.vue:12]: `grow` and `relative` are useless then

File: src/views/HomeView.vue
Comment (fgravin) [reply src/views/HomeView.vue:8]: the panel is not full-height

File: src/components/home/InformationPanel.vue
Comment (ismailsunni) [reply src/components/home/InformationPanel.vue:1]: If we use pinia storage, isn't that enough since we use pinia to store the state?

File: src/components/home/InformationPanel.vue
Comment (ismailsunni) [reply src/components/home/InformationPanel.vue:22]: should I address the transition now with some basic transition or should we wait until the final decision? Based on the daily, they are still working on this transition.

Comment (fgravin) [reply src/components/home/InformationPanel.vue:22]: let's wait

File: src/components/home/InformationPanel.vue
Comment (fgravin) [reply src/components/home/InformationPanel.vue:1]: no, cause here you will have to duplicate the behavior for all pages

**Pull request merged**
